lambda/README.md

```diff
@@ -1,14 +1,21 @@
 # Running a lambda on localstack
 
-1. Create a zip
+## Dependencies
+
+1. [Rialto-api / localstack](https://github.com/sul-dlss-labs/rialto-api/)
+
+## Create the lambda zip file, upload and subscribe
+
 ```
-GOOS=linux go build -o main
-zip lambda.zip main
+make
 ```
 
-1. Start localstack. If you're on a Mac, ensure you are running the docker daemon.
+## Manually run the lambda setup
+
+1. Create a zip
 ```
-SERVICES=lambda,es,sns LAMBDA_EXECUTOR=docker localstack start
```

Comment:
Why are we removing these directions?

Reply:
Because these are presumably already started in the rialto-api docker-compose which I believe I notate in the dependencies.